frontend/src/routes/llm-settings.tsx

```diff
@@ -279,13 +297,35 @@ function LlmSettingsScreen() {
               className="flex flex-col gap-6"
             >
               {!isLoading && !isFetching && (
-                <ModelSelector
-                  models={modelsAndProviders}
-                  currentModel={
-                    settings.LLM_MODEL || "anthropic/claude-sonnet-4-20250514"
-                  }
-                  onChange={handleModelIsDirty}
-                />
+                <>
+                  <ModelSelector
+                    models={modelsAndProviders}
+                    currentModel={
+                      settings.LLM_MODEL || "openhands/claude-sonnet-4-20250514"
+                    }
+                    onChange={handleModelIsDirty}
+                  />
+                  {(settings.LLM_MODEL?.startsWith("openhands/") ||
+                    currentSelectedModel?.startsWith("openhands/")) && (
+                    <div className="mt-2 text-sm text-blue-400">
+                      <Trans
+                        i18nKey={I18nKey.SETTINGS$OPENHANDS_API_KEY_HELP}
+                        components={{
+                          a: (
+                            <a
+                              href="https://app.all-hands.dev/settings/api-keys"
+                              target="_blank"
+                              rel="noopener noreferrer"
+                              className="text-blue-400 hover:underline"
+                            >
+                              {/* Content will be filled by Trans component */}
+                            </a>
+                          ),
+                        }}
+                      />
+                    </div>
+                  )}
+                </>
```

Comment:
@OpenHands the link added here is NOT working. Can you simplify it and maybe add it else where? can we directly add link in translation.json?